.gitignore

```diff
@@ -3,3 +3,4 @@
 .pub
 packages
 pubspec.lock
+.vscode/launch.json
```

Comment:
Please remove this line. It should be in `~/.gitignore_global`

Reply:
Amended as requested.